Example/AppDelegate+CarPlay.swift

```diff
@@ -237,6 +243,164 @@ extension AppDelegate: CarPlaySearchControllerDelegate {
     func popTemplate(animated: Bool) {
         carPlayManager.interfaceController?.popTemplate(animated: animated)
     }
+    
+    func forwardGeocodeOptions(_ searchText: String) -> ForwardGeocodeOptions {
+        let options = ForwardGeocodeOptions(query: searchText)
+        options.focalLocation = AppDelegate.coarseLocationManager.location
+        options.locale = Locale.autoupdatingCurrent.languageCode == "en" ? nil : .autoupdatingCurrent
+        var allScopes: PlacemarkScope = .all
+        allScopes.remove(.postalCode)
+        options.allowedScopes = allScopes
+        options.maximumResultCount = MaximumSearchResults.extended
+        options.includesRoutableLocations = true
+        
+        return options
+    }
+    
+    func recentSearches(with searchText: String) -> [CPListItem] {
```

Comment:
This method was previously exposed as part of `CarPlaySearchControllerDelegate`, since it's not really used on SDK level I've decided to remove it from there, and it's used only on application level.